Add branch for focal as 'debian' is now on groovy.

kgilmer opened the pull request:
This change specifies that a package build of `regolith-desktop` should happen on the focal branch.  As a rule, the `debian` branch tracks the current release version of Ubuntu.

kgilmer committed:
Add branch for focal as 'debian' is now on groovy.

moritzheiber commented:
Are we going to have two separate ISO files then? I'm trying to get idea of the impact of this change.

kgilmer commented:
This change has no bearing on the number of ISO files.  It's more of a house keeping chore to maintain a version of the `regolith-desktop` package for each supported version of Ubuntu.

Most of our packages are able to be identical regardless of Ubuntu version, and so in the PPA the version string for the package is the same across bionic, focal, and groovy.  A few packages need to be different (mainly due to backwards compat breaking changes in gnome programs) per Ubuntu version.  `regolith-desktop` is one of them.

This JSON file is read by [the build.sh script](https://github.com/regolith-linux/regolith-builder/blob/master/build.sh).  It maps every Regolith package to it's git repo, branch, etc.  Without this change the build script cannot generate a `focal` version of `regolith-desktop`.

I added a rule to the dev policy regarding `debian` tracking latest Ubuntu version is here: https://github.com/regolith-linux/regolith-system/wiki/Regolith-Development-Workflow-Summary#r2---git-branch-names-for-ubuntu-versions

(It didn't exist, I just added it)

kgilmer commented:
Regarding ISO files, I have only been creating new ISO files for the latest Ubuntu version + the last LTS release.  If we continue this, then yes this change would also be required to produce distinct ISOs for both Groovy and Focal.
